crates/escrow/src/lib.rs

```diff
@@ -173,6 +176,11 @@ pub mod pallet {
             who: T::AccountId,
             amount: BalanceOf<T>,
         },
+        StakeUpdated {
+            who: T::AccountId,
+            initial_stake: BalanceOf<T>,
+            final_stake: BalanceOf<T>,
+        },
```

Comment:
I don't think we need this since the rewards pallet will already emit events.

Reply:
OK